README.md

```diff
@@ -78,6 +78,21 @@ Deployment on [Linode](https://www.linode.com/) is performed via StackScripts, w
 
 Please submit an [issue](https://github.com/simplex-chat/simplexmq/issues) if any problems occur.
 
+[<img alt="linode" src="./img/digitalocean.png" align="right" width="300">](https://cloud.linode.com/stackscripts/748014)
+
+## Deploy SMP server on DigitalOcean
+
+[DigitalOcean](https://cloud.digitalocean.com) operates on the concept of Droplets. Droplet is a unit of deployment instantiated based on a Snapshot. You can deploy SMP server using provided [SimpleX Server Snapshot](https://marketplace.digitalocean.com/apps/simplex-server):
+
+- Create a DigitalOcean account or login with an already existing one.
+- Create SimpleX server Droplet from Snapshot.
+- Choose the region and plan according to your requirements (cheapest Regular plan should be sufficient).
+- Provide [ssh key](https://www.digitalocean.com/community/tutorials/how-to-set-up-ssh-keys-2) and confirm Droplet creation. 
```

Comment:
Is it optional?

Reply:
No, you won't be able to get public key hash without ssh

Reply:
I meant whether it is optional to provide the key or it can be generated by DO (or you can use key you have on the account). I definitely do not provide a key for each linode I deploy, it's usually pre-selected.

Reply:
You need to provide public key that is present on the OS that you'll be connecting to the Droplet from. DO doesn't provide any ways to put it there for you. I'm also removing instruction, because it is already linked by DO when you click 'new SSH key'.
![image](https://user-images.githubusercontent.com/39944650/125243828-43f71d80-e2f7-11eb-8c5b-6e216323708b.png)